graphql_api/tests/test_owner.py

```diff
@@ -548,3 +549,23 @@ def test_owner_available_plans(self):
             {"planName": "users-pr-inappm"},
             {"planName": "users-pr-inappy"},
         ]
+
+    def test_owner_query_with_no_service(self):
+        current_org = OwnerFactory(
+            username="random-plan-user",
+            service="github",
+        )
+        query = """{
+            owner(username: "%s") {
+                username
+            }
+        }
+        """ % (
+            current_org.username
+        )
+
+        try:
+            self.gql_request(query, provider="", owner=current_org)
+        except MissingService as e:
+            assert str(e) == "Missing service"
+            raise
```

Comment:
If this test is passing then maybe this error is not being rased as expected?

Reply:
why?

Reply:
I think the test would error if this exception was indeed being raised

Reply:
are you talking about raise? it is my understanding that the error will only be raised if the assert condition is not met